packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -414,10 +414,11 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
   Future<bool> isMarkerInfoWindowShown(
     MarkerId markerId, {
     required int mapId,
-  }) {
+  }) async {
     assert(markerId != null);
-    return channel(mapId).invokeMethod<bool>('markers#isInfoWindowShown',
-        <String, String>{'markerId': markerId.value}) as Future<bool>;
+    bool? res  = await channel(mapId).invokeMethod<bool?>('markers#isInfoWindowShown',
+        <String, String>{'markerId': markerId.value}) ;
```

Comment:
Why the intermediate value, vs `return (await ...)!`?

Reply:
You're right, I didn't think of wrapping with () and forcing the result at the end with !